collision_radius: 0.1
left_arm:
  chain_end: left_forearm_roll_link
  chain_start: head_base_link
  joint_names:
  - left_shoul_base2shoul_joint
  - left_shoul2shoul_rot_joint
  - left_arm2armrot_joint
  - left_armrot2elbow_joint
  - left_forearm2forearmrot_joint
  - left_forearmrot2forearm_pitch_joint
  - left_forearm_pitch2forearm_roll_joint
num_waypoints: 50
right_arm:
  chain_end: right_forearm_roll_link
  chain_start: head_base_link
  joint_names:
  - right_shoul_base2shoul_joint
  - right_shoul2shoul_rot_joint
  - right_arm2armrot_joint
  - right_armrot2elbow_joint
  - right_forearm2forearmrot_joint
  - right_forearmrot2forearm_pitch_joint
  - right_forearm_pitch2forearm_roll_joint
time: 1.0
urdf_file: package://rx1_description/urdf/rx1.urdf
